src/MdOpts.h

```diff
@@ -0,0 +1,30 @@
+#ifndef INC_MDOPTS_H
+#define INC_MDOPTS_H
+#include "Constraints.h"
+class MdOpts {
+  public:
+    MdOpts();
+
+    int GetOptsFromArgs(ArgList&);
+
+    void PrintOpts() const;
+
+    Constraints::ShakeType Shake() const { return shakeType_; }
+    double ScaleEE()       const { return scaleEE_; }
+    double ScaleNB()       const { return scaleNB_; }
```

Comment:
Do you support mixed scaling like for glycam-amber combos?

Reply:
No, right now OpenMM support is very basic, so only the universal scaling is supported :-(